frontend/pages/hosts/ManageHostsPage/HostTableConfig.tsx

```diff
@@ -424,7 +425,40 @@ const allHostTableHeaders: IDataColumn[] = [
       />
     ),
     accessor: "public_ip",
-    Cell: (cellProps: ICellProps) => <TextCell value={cellProps.cell.value} />,
+    Cell: (cellProps: ICellProps) => {
+      if (cellProps.cell.value) {
+        return <TextCell value={cellProps.cell.value} />;
+      }
+      return (
+        <>
+          <span
+            className="text-cell text-muted tooltip"
+            data-tip
+            data-for={"public-ip-tooltip"}
+          >
+            ---
```

Comment:
@roperzh just merged a nice [setup to use a `DEFAULT_EMPTY_CELL`](https://github.com/fleetdm/fleet/pull/9899#pullrequestreview-1303975381) constant, might be good to use that here instead of hardcoded '---'